src/client/menu/menu.c

```diff
@@ -283,12 +287,18 @@ Key_GetMenuKey(int key)
 		case K_BACKSPACE:
 		case K_DEL:
 		case K_KP_DEL:
+                    if (IN_NumpadIsOn() == true) { break; }
```

Comment:
Not 100% sure but it seems like the indention is wrong.

Reply:
Weird one, displays correctly in the editors, Notepad, Sublime Text, Visual Studio, but tabs are broken when posting to GitHub. Very annoying. Vim resolved this.